pkg/ddl/modify_column_test.go

```diff
@@ -607,3 +607,51 @@ func TestModifyColumnWithIndexesWriteConflict(t *testing.T) {
 		"3 3 3 c",
 		"4 4 4 d"))
 }
+
+func TestAdminAlterModifyColumnDDLJob(t *testing.T) {
```

Comment:
seems this test and the case in `pkg/ddl/index_change_test.go` are so similiar, maybe unified into one, with different param